apps/language_server/.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.11.3-otp-23
```

Comment:
some time ago we resigned from using .tool-versions

Reply:
Oh, this shouldn't have been commited! I ended up setting local versions to fix a broken test between < 1.10 and >= 1.10

I'll revert this.

Reply:
Done :)